html/Minimo/style.css

```diff
@@ -0,0 +1,258 @@
+html {
+  font-size: 2rem;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+h1, h2, h3, h4, h5, h6, p, ul {
+  margin: 0;
+}
+
+/* Font styles */
+.title--font {
+  font-family: "PlayfairDisplay";
+  font-size: 0.93rem;
+  font-weight: normal;
+}
+
+.small-text--font {
+  font-family: Ubuntu;
+  font-size: 0.45rem;
+  font-weight: 300;
+}
+
+.content-text--font {
+  font-family: "PlayfairDisplay";
+  font-size: 0.45rem;
+  font-weight: normal;
+}
+
+/* Container */
+.container {
+  padding: 0 70px;
+}
+
+/* Navbar */
+.navbar {
+  display: flex;
+  align-items: center;
+  margin-top: 35px;
+}
+
+.navbar__logo {
+  flex: 1;
+  font-family: "Inconsolata", monospace;
+  font-size: 0.93rem;
+  letter-spacing: 3px;
+  text-transform: uppercase;
+}
+
+.navbar__menu-list {
+  display: none;
+}
+
+.navbar__menu-list .menu__item {
+  color: #626262;
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+.navbar__hamburger-icon {
+  cursor: pointer;
+}
+
+/* Banner image */
+.banner {
+  width: 100%;
+  margin-top: 36px;
+}
+
+.content-container {
+  margin: 50px 0 61px 0;
+}
+
+/* Articles */
+.articles {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-row-gap: 100px;
+  margin-top: 120px;
+}
+
+.article__img {
+  width: 100%;
+}
+
+.article__type {
+  margin-top: 32px;
+  color: #b4ad9e;
+  text-transform: uppercase;
+}
+
+.article__title {
+  margin-top: 28px;
+  color: #626262;
+}
+
+.article__content {
+  margin-top: 34px;
+  color: #626262;
+  line-height: 1.71;
+}
+
+.article__comment {
+  margin-top: 52px;
+  color: #b4ad9e;
+  text-transform: uppercase;
+}
+
+/* Newsletter */
+.newsletter {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 143.5px;
```

Comment:
height? maybe set paddings and let content set height

Reply:
FYI: try to avoid fractional pixels. Browser rounds it anyway